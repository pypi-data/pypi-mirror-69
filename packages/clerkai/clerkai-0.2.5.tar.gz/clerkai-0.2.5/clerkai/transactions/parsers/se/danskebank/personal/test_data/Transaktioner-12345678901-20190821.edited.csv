"Bokföringsdag";"Specifikation";"Belopp";"Saldo";"Status";"Avstämt"
"2019-08-21";"CAFFE NERO K25";"-115,00";"4.558,07";"Utförd";"Nej"
"2019-08-20";"Swish till Herr Nilsson";"-2.500,00";"5.141,43";"Utförd";"Nej"
"2019-08-20";"Swish från Tant Grön";"780,00";"7.641,43";"Utförd";"Nej"
"2019-08-20";"Swish från Farbror Blå";"200,00";"6.861,43";"Utförd";"Nej"
"2019-08-20";"7-ELEVEN 12345";"-104,00";"6.661,43";"Utförd";"Nej"
"2019-08-20";"Taxi Kurir I Stockho";"-120,00";"9.065,43";"Utförd";"Nej"