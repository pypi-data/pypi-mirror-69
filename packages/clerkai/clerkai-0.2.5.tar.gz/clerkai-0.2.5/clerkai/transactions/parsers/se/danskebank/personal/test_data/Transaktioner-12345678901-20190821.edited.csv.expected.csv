Real Date,Bank Date,Payee,Bank Message,Amount,Balance,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Balance
,2019-08-21,,CAFFE NERO K25,-115.00,4558.07,"{'Bokföringsdag': '2019-08-21', 'Specifikation': 'CAFFE NERO K25', 'Belopp': -115.0, 'Saldo': 4558.07, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-21,,CAFFE NERO K25,-115.0,4558.07
,2019-08-20,,Swish till Herr Nilsson,-2500.00,5141.43,"{'Bokföringsdag': '2019-08-20', 'Specifikation': 'Swish till Herr Nilsson', 'Belopp': -2500.0, 'Saldo': 5141.43, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-20,,Swish till Herr Nilsson,-2500.0,5141.43
,2019-08-20,,Swish från Tant Grön,780.00,7641.43,"{'Bokföringsdag': '2019-08-20', 'Specifikation': 'Swish från Tant Grön', 'Belopp': 780.0, 'Saldo': 7641.43, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-20,,Swish från Tant Grön,780.0,7641.43
,2019-08-20,,Swish från Farbror Blå,200.00,6861.43,"{'Bokföringsdag': '2019-08-20', 'Specifikation': 'Swish från Farbror Blå', 'Belopp': 200.0, 'Saldo': 6861.43, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-20,,Swish från Farbror Blå,200.0,6861.43
,2019-08-20,,7-ELEVEN 12345,-104.00,6661.43,"{'Bokföringsdag': '2019-08-20', 'Specifikation': '7-ELEVEN 12345', 'Belopp': -104.0, 'Saldo': 6661.43, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-20,,7-ELEVEN 12345,-104.0,6661.43
,2019-08-20,,Taxi Kurir I Stockho,-120.00,9065.43,"{'Bokföringsdag': '2019-08-20', 'Specifikation': 'Taxi Kurir I Stockho', 'Belopp': -120.0, 'Saldo': 9065.43, 'Status': 'Utförd', 'Avstämt': 'Nej'}",,2019-08-20,,Taxi Kurir I Stockho,-120.0,9065.43
